Whoisthat/FaceInformationForm.cs

```diff
@@ -54,12 +76,56 @@ private void BtnSave_Click(object sender, EventArgs e)
             Close();
         }
 
+        private bool Upload(byte[] sample,string name)
+        {
+            string url = "http://localhost:42069/api/Post";
+            bool _fileUploaded = false;
+            try
+            {
+                
+                HttpClient httpClient = new HttpClient();
+                MultipartFormDataContent content = new MultipartFormDataContent();
+                ByteArrayContent byteContent = new ByteArrayContent(sample);
+                content.Add(byteContent, name);
+                Task taskUpload = httpClient.PostAsync(url, content).ContinueWith(task =>
+                {
+                    if (task.Status == TaskStatus.RanToCompletion)
+                    {
+                        var response = task.Result;
+                        if (response.IsSuccessStatusCode)
+                        {
+                            _fileUploaded = true;
+                        }
+                    }
+
+                });
+                taskUpload.Wait();
+                if (_fileUploaded)
+                httpClient.Dispose();
+            }catch (Exception ex)
+            {
+                _fileUploaded = false;
+                throw new Exception("File not uploaded");
+            }
+           return _fileUploaded;
+       
+
+        }
+
         private byte[] ToByteArray(Image<Gray, byte> image)
         {
             MemoryStream ms = new MemoryStream();
             new Bitmap(image.ToBitmap()).Save(ms, ImageFormat.Bmp);
             byte[] header = ms.ToArray();
             return header;
         }
+
+        private void btnRefresh_Click(object sender, EventArgs e)
+        {
+        }
```

Comment:
Does this empty method have any purpose? Maybe add a comment explaining that at least?
```suggestion
            // here will be code to get training data form server
        }
```